app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Users::SessionsController < Devise::SessionsController
+  before_filter :set_flash, only: [:new]
+
+  private
+
+  def set_flash
+    @flash = Notifier.new flash
```

Comment:
@vlasiak, the naming looks great, but I suggest to use `FlashNotification` term, as  `Notifier` seems to perform some actions related to notification (sending, pushing, etc), but the class actually represents a simple notification, so `FlashNotification` would be OK.